recipes/r-sctransform/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set version = '0.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sctransform
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sctransform_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sctransform/sctransform_{{ version }}.tar.gz
+  sha256: d7f4c7958693823454f1426b23b0e1e9c207ad61a7a228602a1885a1318eb3e4
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - {{native}}toolchain        # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-mass
+    - r-matrix
+    - r-rcpp
+    - r-rcppeigen
+    - r-future
+    - r-future.apply
+    - r-ggplot2
+    - r-gridextra
+    - r-reshape2
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-mass
+    - r-matrix
+    - r-rcpp
+    - r-rcppeigen
+    - r-future
+    - r-future.apply
+    - r-ggplot2
+    - r-gridextra
+    - r-reshape2
+
+test:
+  commands:
+    - $R -e "library('sctransform')"           # [not win]
+    - "\"%R%\" -e \"library('sctransform')\""  # [win]
+
+about:
+  home: https://github.com/ChristophH/sctransform
+  license: GPL-3
```

Comment:
the license_file is missing here, but we need it because its GPL :(

Reply:
Haven't we always just ignored that? Anyway I'll add it.

Reply:
I think it was removed from the skeleton since so few packages actually ship the license. Anyway, it's added now.

Reply:
Look at the checklist above ;)